src/app/containers/ATIAnalytics/atiUrl/index.js

```diff
@@ -143,6 +145,20 @@ export const buildATIPageTrackPath = ({
       value: ldpThingIds,
       wrap: true,
     },
+    {
+      key: 'x16',
+      description: 'campaigns',
+      value: (Array.isArray(campaigns) ? campaigns : [])
+        .map(campaign => campaign.campaignName.replace(/ /g, '%20'))
```

Comment:
Could be more readable if `/ /g` was:

`const spaceRegex = / /g` then `.replace(spaceRegex, '%20')`